SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBTrigPrimTestAlgo.cc

```diff
@@ -243,81 +250,31 @@ void EcalEBTrigPrimTestAlgo::run(const edm::EventSetup & setup,
 	    resultTcp.push_back(tp);
 	  
 
+	} // Loop over the xStals in  a sztrip
 
-	} // Loop over the xStals
 
+      } //loop over strips in one tower
 
 
 
-      }//loop over strips in one tower
+
+  } // loop over the towers
   
 
 
 
-  }
+}
 
 
 
 
-  /*
-  for (unsigned int i=0;i<digi->size();i++) {
-    EBDataFrame myFrame((*digi)[i]);  
-    const EBDetId & myid1 = myFrame.id();
-    tp=  EcalTriggerPrimitiveDigi(  myid1);   
-    tp.setSize( myFrame.size());
-    int nSam=0;
-
-    if (debug_) {
-      std::cout << " data frame size " << myFrame.size() << " Id " <<  myFrame.id()  << std::endl;
-      std::cout << " Sample data ADC: " << std::endl;
-      for (int iSample=0; iSample<myFrame.size(); iSample++) {
-	std::cout << " " << std::dec<< myFrame.sample(iSample).adc() ;
-      }
-      std::cout<<std::endl;
-    }
-
-    
-    this->getLinearizer(i)->setParameters( myFrame.id().rawId(),ecaltpPed_,ecaltpLin_,ecaltpgBadX_) ; 
-    //this->getLinearizer(i)->process( myFrame,lin_out_[i]);
-
-    if (debug_) {
-      std::cout<< "cryst: "<< i <<"  value : "<<std::dec<<std::endl;
-      std::cout<<" lin_out[i].size()= "<<std::dec<<lin_out_[i].size()<<std::endl;
-      for (unsigned int j =0; j<lin_out_[i].size();j++){
-	std::cout <<" "<<std::dec<<(lin_out_[i])[j];
-      }
-      std::cout<<std::endl;
-    }
-
-
-    for (int iSample=0; iSample<myFrame.size(); iSample++) {
-      etInADC= myFrame.sample(iSample).adc();
-      EcalEBTriggerPrimitiveSample mysam(etInADC);
-      tp.setSample(nSam, mysam );
-      nSam++;
-      if (debug_) std::cout << "in TestAlgo" <<" tp size "<<tp.size() << std::endl;
-    }
-
-    if (!tcpFormat_)
-      result.push_back(tp);
-    else 
-      resultTcp.push_back(tp);
-    
-   
-    if (debug_) std::cout << " result size " << result.size() << std::endl;
-    
-    
-    
-  }
-  */
 
-}
 
 
   
 
 
-
+/*
```

Comment:
delete commented-out code

Reply:
Please do, or justify why you prefer not doing it